sources/packages/backend/apps/api/src/auth/decorators/institution/institution-student-data-access.decorator.ts

```diff
@@ -10,5 +10,11 @@ export const INSTITUTION_HAS_STUDENT_DATA_ACCESS_KEY =
  * Provide context that it's consumer must be
  * validated to have access to data of given student.
  */
-export const HasStudentDataAccess = (studentIdParamName: string) =>
-  SetMetadata(INSTITUTION_HAS_STUDENT_DATA_ACCESS_KEY, studentIdParamName);
+export const HasStudentDataAccess = (
+  studentIdParamName: string,
+  applicationIdParamName?: string,
+) =>
+  SetMetadata(INSTITUTION_HAS_STUDENT_DATA_ACCESS_KEY, [
+    studentIdParamName,
```

Comment:
Yesterday post stand-up, I believe we decided to go with the approach of using institutionId in the query.
when did that change?